cli/migrate.go

```diff
@@ -0,0 +1,59 @@
+// Copyright © 2020 Solus Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"github.com/DataDrake/cli-ng/cmd"
+	log "github.com/DataDrake/waterlog"
+	"github.com/DataDrake/waterlog/level"
+	"github.com/getsolus/qol-assist/core"
+	"os"
+)
+
+var migrate = &cmd.CMD{
+	Name:  "migrate",
+	Short: "Applies migrations that are available on the system",
+	Alias: "m",
+	Args:  &struct{}{},
```

Comment:
This should be able to just be unassigned. If I forgot, please file an issue with the `cli-ng` repo so that I fix it.

Reply:
https://github.com/DataDrake/cli-ng/issues/11